system_controller/config/config.yaml

```diff
@@ -6,7 +6,7 @@
 
     # Long: Timeout for each service to be detected as available in milliseconds
     # Units: milliseconds
-    service_timeout_ms : 200
+    service_timeout_ms : 18000
```

Comment:
Was this change meant to be committed to develop? 18 seconds seems too large for the timeout

Reply:
18000 is a very arbitrary value assigned during testing. This relates to specific Swarm behavior of how it deploys services differently (and a bit slower) than our normal Docker deployment. I can do some testing to see what this value actually needs to be, it should be lower but unsure by how much. I can show you a demo sometime as an example.